src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindEvents();
+    }
+
+    /**
+     * Triggers binded event with a given value.
```

Comment:
Binds event listeners to digit buttons.

This functions does not trigger, it just binds.